components/gitpod-cli/cmd/tasks-list.go

```diff
@@ -48,11 +49,36 @@ var listTasksCmd = &cobra.Command{
 			2: tablewriter.FgHiBlackColor,
 		}
 
+		mapCurrentToColor := map[bool]int{
+			false: tablewriter.FgWhiteColor,
+			true:  tablewriter.FgHiGreenColor,
+		}
+
 		for _, task := range tasks {
 			colors := []tablewriter.Colors{}
 
+			ppid := int64(os.Getppid())
+
+			isCurrent := false
+
+			if task.State == 1 {
```

Comment:
Can we use a proper name instead of just a number, I think this is the correct way
```suggestion
			if task.State == api.TaskState_running {
```

Reply:
Done!